### PR TITLE
Log out of all sessions except the current session

### PR DESCRIPTION
**Pull request by exu3**
https://hackclub.slack.com/archives/CN523HLKW/p1646405305155659

Screen Shot 2022-03-05 at 02 25 30

---

This PR:

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def logout
 
   def logout_all
     sign_out_of_all_sessions
-    redirect_to root_path
+    redirect_to edit_user_path(current_user), flash: { success: "Success" }
```
Comment (garyhtou) [inline app/controllers/users_controller.rb]: love this!

Comment (garyhtou) [reply app/controllers/users_controller.rb]: This isn't really related to this pr, but could you add a flash to `logout_session`'s redirect too?

**Review by garyhtou — APPROVED** [review]
Good work here!

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (exu3) [reply app/controllers/users_controller.rb]: Screen Shot 2022-03-07 at 13 37 52

I think there's already a flash

Comment (garyhtou) [reply app/controllers/users_controller.rb]: whoops! you're right! I see it on line 94 now. thanks Ella!

**Pull request merged**